tests/test.yml

```diff
@@ -0,0 +1,73 @@
+---
+
+- name: Test sansible.vernemq Role
+  hosts: all
+
+  vars:
+    vernemq:
+      version: 1.2.0
+      nofile: 65536
+      configuration:
+        leveldb.maximum_memory.percent: 8
+        plugins.vmq_plugin: on
+
+  pre_tasks:
+    - name: Update apt
+      become: yes
+      apt:
+        cache_valid_time: 1800
+        update_cache: yes
+      tags:
+        - build
+
+  roles:
+    - role: sansible.vernemq
+      sansible_vernemq: "{{ vernemq }}"
+      tags:
+        - build
+        - configure
+
+  post_tasks:
+    - name: Assert VerneMQ is running
+      become: yes
+      command: vernemq getpid
+      changed_when: no
+      register: vernemq_pid
+      tags:
+        - assert
```

Comment:
You can use the service module like so:

```YAML
    - name: Assert VerneMQ is running
      become: yes
      service:
        name: vernemq
        state: started
      register: service_start_check
      failed_when: service_start_check.changed
      tags:
        - assert
```

Reply:
The reason I do it this way is to get the PID for VerneMQ, which is required for "Get nofiles for running VerneMQ Process" further down.

Reply:
Fair enough